recipe/meta.yaml

```diff
@@ -69,6 +70,7 @@ test:
     - python -c 'import regrid2'
     - python -c 'import regrid2._regrid'
     - python -c 'import regrid2._scrip'
+    - python -c 'from regrid2 import ESMFRegrid'
```

Comment:
@chengzhuzhang, hopefully this check will prevent issues like https://github.com/E3SM-Project/e3sm_diags/issues/711 from happening in the future.

Reply:
It looks like regridding behavior gets back to normal with this fix!